lib/dogstatsd.rb

```diff
@@ -340,6 +372,9 @@ def send_stats(stat, delta, type, opts={})
     if sample_rate == 1 or rand < sample_rate
       # Replace Ruby module scoping with '.' and reserved chars (: | @) with underscores.
       stat = stat.to_s.gsub('::', '.').tr(':|@', '_')
+      if opts[:manual_sample_rate]
+        sample_rate *= opts[:manual_sample_rate]
```

Comment:
I think that's a good idea. When I read it originally my eyes missed the `*=` and it wasn't until later in reading the test that I realized that it was the product. Good idea!